docs/source/developer/repomap.rst

```diff
@@ -0,0 +1,447 @@
+A Map of the Numba Repository
+=============================
+
+The Numba repository is quite large, and due to age has functionality spread
+around many locations.  To help orient developers, this document will try to
+summarize where different categories of functionality can be found.
+
+.. note::
+    It is likely that the organization of the code base will change in the
+    future to improve organization.  Follow `issue #3807 <https://github.com/numba/numba/issues/3807>`_
+    for more details.
+
+
+Support Files
+-------------
+
+Build and Packaging
+'''''''''''''''''''
+
+- ``setup.py`` - Standard Python distutils/setuptools script
+- ``MANIFEST.in`` - Distutils packaging instructions
+- ``requirements.txt`` - Pip package requirements, not used by conda
+- ``versioneer.py`` - Handles automatic setting of version in installed package from git tags
+- ``.flake8`` - Preferences for code formatting.  Files should be fixed and removed from exception list as time allows.
+- ``buildscripts/condarecipe.local`` - Conda build recipe
+- ``buildscripts/remove_unwanted_files.py`` - Helper script to clean up files.  Used by build recipes.
+- ``buildscripts/condarecipe_clone_icc_rt`` - Recipe to build a standalone icc_rt package.  Not used anymore?
```

Comment:
May need regular re-runs against newer ICC RT's from Intel? There's no `icc_rt` package on defaults for non-Windows platforms (Windows has it due to Fortran? compilers).

Reply:
In fact, we do, there was a patch to update last week!